### PR TITLE
docs: add examples for assuming roles from LINZ accounts

### PR DESCRIPTION
**Pull request by billgeo**
Better documentation with examples to support #1585

## Reference

[Code review checklist](CODING.md#Code-review-checklist)

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -106,15 +106,48 @@ Geostore allows read/write access for users authorized by SAML identity provider
...
+- Add Geostore roles to assume to AWS config
+
+  ```bash
+  echo "[profile geostore-api-users]
+  role_arn=arn:aws:iam::632223577832:role/nonprod-api-users
+  source_profile=li-small-apps-nonprod
+
+  [profile geostore-s3-users]
+  role_arn=arn:aws:iam::632223577832:role/nonprod-s3-users
+  source_profile=li-small-apps-nonprod
```
Comment (l0b0) [inline USAGE.md:131]: Presumably this assumes you already have a profile named "li-small-apps-nonprod".

Comment (billgeo) [reply USAGE.md:131]: Which should be the case for LI people. Is there some better way to indicate that this is an example?
Should I replace `li-small-apps-nonprod` with `<ANY_LINZ_AWS_ACCOUNT_PROFILE>` or something like that. Sorry not sure what teh documentation conventions are.

Comment (l0b0) [reply USAGE.md:131]: There's plenty of [conventions](https://xkcd.com/927/) :wink: - we can just leave this as is if they are expected to already have the relevant profile. Alternatively, we could refer them to the [instructions to set all their profiles in one go](https://confluence.linz.govt.nz/display/GEOD/Login+to+AWS+Service+Accounts+via+Azure+in+Command+Line). Would that work?

Comment (billgeo) [reply USAGE.md:131]: Yeah. I thought about the confluence pages, but they're not public. So not really ideal.

Comment (billgeo) [reply USAGE.md]: Added an example of how to set a variable with an example source profile. Hopefully that's a bit clearer.

File: USAGE.md [hunk 2]
```diff
@@ -106,15 +106,44 @@ Geostore allows read/write access for users authorized by SAML identity provider
 
 Example of AWS service account authentication and authorization in to Geostore users role via Azure:
 
-- Log in to Geostore AWS account using Geostore users role
+- Log in to Geostore AWS account
 
   ```bash
-  aws-azure-login --profile geostore-users
+  aws-azure-login --profile li-geostore-prod
```
[the remark this answers is not in the document]
Comment (billgeo) [reply USAGE.md]: Done

**Review by l0b0 — APPROVED** [review]

**Pull request merged**